From e7c9888197491747a98f413b9b3503e9fb42b26d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Mon, 25 May 2020 13:08:24 +0300
Subject: [PATCH] gtk: Skip building the metainfo xml when targeting windows

This file is only supposed to be used by desktop managers to get info
about the app. On windows, the file is unused.

Since 368576dc9aa94c792a309e227c9a893776d8cdd1, the translation step
for this file requires gettext 0.20 or newer.
---
 gtk/src/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gtk/src/Makefile.am b/gtk/src/Makefile.am
index 147df424e..f9e30458f 100644
--- a/gtk/src/Makefile.am
+++ b/gtk/src/Makefile.am
@@ -41,8 +41,10 @@ hb_menu = fr.handbrake.ghb.desktop
 
 metainfodir = $(datarootdir)/metainfo
 metainfo_in_files = fr.handbrake.ghb.metainfo.xml.in
+if MINGW
 metainfo_DATA = $(metainfo_in_files:.xml.in=.xml)
 dist_metainfo_DATA = $(metainfo_files)
+endif
 
 EXTRA_DIST = \
 	$(metainfo_in_files)

